allocations/templates/allocations/approval.html

```diff
@@ -31,7 +31,7 @@ <h3 class="modal-title">Allocation Approval</h3>
             <tr ng-show="data.allocation.computeRequested && data.allocation.computeRequested > 0 && data.type == 'approve'">
                 <td><strong>Compute Allocated:</strong>&nbsp;<sup><i style="color: red;font-size:0.5em;" class="fa fa-asterisk"></i></sup></td>
                 <td>
-                    <input type="number" name="computeAllocated" ng-model="data.allocation.computeAllocated" ng-required="true" value=''>
+                    <input type="number" name="computeAllocated" ng-model="data.allocation.computeAllocated" ng-required="data.type == 'approve'" value=''>
```

Comment:
What is this change doing? It was a bit hard to follow how this `type` comes in here

Reply:
It's defined in the controller https://github.com/ChameleonCloud/portal/blob/master/allocations/scripts/controllers/controller.js#L130-L134. Type can be either "approve" or "reject". And if you click "reject", some fields will be hide. See the screenshot below.

<img width="1623" alt="Screen Shot 2020-06-02 at 12 37 15 PM" src="https://user-images.githubusercontent.com/8900716/83559595-715d7f00-a4da-11ea-9be5-cc97fb31bb26.png">

Before fix, the fields like "compute allocated" will be hidden, but still required, which makes the "submit" button unable to click. So whether the field is required or not depends on the "type" as well.